pygmsh/built_in/geometry.py

```diff
@@ -192,6 +192,27 @@ def add_physical_volume(self, volumes, label=None):
         self._add_physical('Volume', volumes, label=label)
         return
 
+    def add_transfinite_lines(self, lines, size):
+        self._GMSH_CODE.append(
+            'Transfinite Line {{{0}}} = {1};'.format(', '.join([l.id for l in lines]), size
+            ))
+
+    def add_transfinite_surface(self, surface, size=None):
+        assert surface.num_edges == 4, \
+            'a transfinite surface can only have 4 sides'
+        if size is not None:
+            assert isinstance(surface, (PlaneSurface, Surface, self.Polygon)), \
+                'we can create transfinite lines only if we have a line loop'
+            self.add_transfinite_lines(
+                [surface.line_loop.lines[0], surface.line_loop.lines[2]],
+                size[0])
+            self.add_transfinite_lines(
+                [surface.line_loop.lines[1], surface.line_loop.lines[3]],
+                size[1])
+        self._GMSH_CODE.append(
+            'Transfinite Surface {{{}}};'.format(surface.id)
+            )
```

Comment:
Style: `return` (same for `add_transfinite_lines`)

All other `add_*` function return an object that can be inspected or used otherwise (see, e.g., the `Polygon` class). It'd be nice if there was something like that for the transfinite entities too.

Reply:
in fact this does not generate a new entity, it rather sets some attributes to existing ones.
should I rename them `set_transfinite_*()`?

Reply:
Yes, that's better. (Also include a `return` regardless.)